app/src/legacy/res/layout/view_config_widget.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorBackgroundLight"
+    android:orientation="vertical">
+
+    <CheckBox
+        android:id="@+id/widget_cv_show_counter"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:checked="true"
+        android:text="@string/widget_show_counter"
+        android:textSize="12sp" />
+
+    <CheckBox
+        android:id="@+id/widget_cv_show_host"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:checked="true"
+        android:text="@string/widget_show_domain"
+        android:textSize="12sp" />
+
+    <CheckBox
+        android:id="@+id/widget_cv_show_dns"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:checked="true"
+        android:text="@string/widget_show_dns"
+        android:textSize="12sp" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/widget_alpha"
+        android:textSize="12sp" />
+
+    <SeekBar
+        android:id="@+id/widget_cv_alpha"
+        android:layout_width="match_parent"
+        android:layout_height="60dp"
+        android:max="255"
+        android:progress="255" />
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="#ff888888">
```

Comment:
Colors should be in an xml file in values/

Reply:
The color was intended as a placeholder. I forgot to replace it. I thought that an image in the background made the effect of the alpha value clearer only had no fitting one. Any ideas what would be suitable?

Reply:
Maybe gradient. Ok leave it as it is, I'll play with it.